front/src/components/celebrities/celebrities.component.js

```diff
@@ -0,0 +1,92 @@
+export default {
+  name: 'ff-celebrities',
+  data: function () {
+    return {
+      celebrities: [
+        {
+          FirstName: 'Vlad',
+          LastName: 'Volkovskyi',
+          src: require('@components/celebrities/actorImg.png'),
+          CountViews: '1m',
+          JobTitle: ['producer', 'actor']
+        },
+        {
+          FirstName: 'Роберт',
+          LastName: 'Дауни (младший)',
+          src: require('@components/celebrities/actorImg.png'),
+          CountViews: '1000k',
+          JobTitle: ['producer', 'actor']
+        },
+        {
+          FirstName: 'Роберт',
+          LastName: 'Дауни (младший)',
+          src: require('@components/celebrities/actorImg.png'),
+          CountViews: '1000k',
+          JobTitle: ['producer', 'actor']
+        },
+        {
+          FirstName: 'Роберт',
+          LastName: 'Дауни (младший)',
+          src: require('@components/celebrities/actorImg.png'),
+          CountViews: '1000k',
+          JobTitle: ['producer', 'actor']
+        },
+        {
+          FirstName: 'Актер',
+          LastName: 'Известный',
+          src: require('@components/celebrities/actorImg.png'),
+          CountViews: '3.02m',
+          JobTitle: ['producer', 'actor', 'writer']
+        },
+        {
+          FirstName: 'Актер',
+          LastName: 'Известный',
+          src: require('@components/celebrities/actorImg.png'),
+          CountViews: '3.5m',
+          JobTitle: ['producer', 'actor', 'writer']
+        },
+        {
+          FirstName: 'Актер',
+          LastName: 'Известный',
+          src: require('@components/celebrities/actorImg.png'),
+          CountViews: '3.5m',
+          JobTitle: ['producer', 'actor', 'writer']
+        },
+        {
+          FirstName: 'Актер',
+          LastName: 'Известный',
+          src: require('@components/celebrities/actorImg.png'),
+          CountViews: '3.5m',
+          JobTitle: ['producer', 'actor', 'writer']
+        },
+        {
+          FirstName: 'Актер',
+          LastName: 'Известный',
+          src: require('@components/celebrities/actorImg.png'),
+          CountViews: '3.5m',
+          JobTitle: ['producer', 'actor', 'writer']
+        }
+      ]
+    };
+  },
+  methods: {
+    modifyElements () {
+      this.celebrities.forEach(elem => { elem.JobTitle = elem.JobTitle.join(', '); });
+    }
+  },
+  created () {
+    this.modifyElements();
+    const rowsCount = Math.floor(this.celebrities.length / 4), row4Columns = [];
+    for (let spliceCount = 0; spliceCount < rowsCount; spliceCount++) {
+      row4Columns.push(this.celebrities.splice(0, 4));
+    }
+    if (this.celebrities.length) {
+      row4Columns.push(this.celebrities);
+    }
+    this.celebrities = row4Columns.sort(
+      function (fstArr, secArr) {
+        return secArr.length - fstArr.length;
+      }
+    );
```

Comment:
why do you need calculate row numbers? it should be resolved by css
and why do you sort data?

Reply:
I need an array(rows) of arrays(columns in row) in order to display it using v-for, like in figma design (4 cards per row)

Reply:
What do you mean by resolving by css ?

Reply:
because it is managed by css: widths, flex properties like flex-wrap or other ways
it should be resolved by css

Reply:
But it's just the structure of template